frontend/src/metabase/palette/hooks/useCommandPalette.tsx

```diff
@@ -242,22 +244,15 @@ export const useCommandPalette = ({
               dispatch(push(href));
             }
           },
-          extra:
-            item.model === "table"
-              ? {
-                  database: item.database.name,
-                  href: Urls.modelToUrl(item),
-                  iconColor: icon.color,
-                }
-              : {
-                  parentCollection:
-                    item.parent_collection.id === null
-                      ? ROOT_COLLECTION.name
-                      : item.parent_collection.name,
-                  isVerified: item.moderated_status === "verified",
-                  href: Urls.modelToUrl(item),
-                  iconColor: icon.color,
-                },
+          extra: {
+            isVerified:
+              item.model === "table"
+                ? false
+                : item.moderated_status === "verified",
```

Comment:
nit: for readability maybe do:

```suggestion
              item.model !== "table" && item.moderated_status === "verified",
```